### ISSUE TITLE
ZeroDivisionError: float division by zero

### ISSUE THREAD

**Issue by bhamehul** [RobinL/fuzzymatcher#42]
I've been trying to match 2 dfs but get the following error.

```
File "fuzzyLookup.py", line 18, in <module>
    linked_table = link_table(app_annie_apps, our_apps, aa_apps_on, our_apps_on)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\__init__.py", line 24, in link_table
    m.match_all()
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\matcher.py", line 92, in match_all
    self.link_table = self._match_processed_data()
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\matcher.py", line 136, in _match_processed_data
    this_record.find_and_score_potential_matches()
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\record.py", line 68, in find_and_score_potential_matches
    self.matcher.data_getter.get_potential_match_ids_from_record(self)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\data_getter_sqlite.py", line 93, in get_potential_match_ids_from_record
    self._search_specific_to_general_single(token_list, rec_left)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\data_getter_sqlite.py", line 119, in _search_specific_to_general_single
    self._add_matches_to_potential_matches(new_matches, rec_left)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\data_getter_sqlite.py", line 167, in _add_matches_to_potential_matches
    scored_potential_match = self.matcher.scorer.score_match(rec_left.record_id, right_id)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\scorer_default.py", line 57, in score_match
    p = self._field_to_prob(f_left, record_left, record_right)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\scorer_default.py", line 77, in _field_to_prob
    prob_unmatching1 = self._get_prob_unmatching(unmatching_tokens_left, tokens_right, field_right, field_left)
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\scorer_default.py", line 100, in _get_prob_unmatching
    if not self._is_misspelling_of_one(umt, record_tokens):
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\scorer_default.py", line 111, in _is_misspelling_of_one
    if self.matcher.token_comparison.is_mispelling(token, t):
  File "C:\Users\bhamehul\AppData\Local\Continuum\Anaconda2\envs\python3\lib\site-packages\fuzzymatcher\tokencomparison.py", line 38, in is_mispelling
    if max(t1f, t2f)/min(t1f, t2f) < self.number_fuzz_threshold:
ZeroDivisionError: float division by zero
```

**Comment by ck090** [comment]
Hey,

We had the same error. It will be resolved by correcting the code in tokencomparison.py file
This file will be in the site-packages folder of which ever python version you used.
To get the path you can type in the command
`python -m site`

Now, change in the code is to add 
`except ZeroDivisionError:
return False`
after line 42 in that file.

**Comment by RobinL** [comment]
Closed by #42

**Comment by kanlancb** [comment]
Hello @ck090 , I too am getting the float error, but didnt understand your fix. I pip installed the package in colab for context

Could you please explain more on how to solve this issue?
